### ISSUE TITLE
rename the "Date Original Version Created" field to "Date Taken"

### ISSUE THREAD

**Issue by hacole01** [AAFC-BICoE/dina-planning#77]
GIVEN I have accessed DINA as manager

WHEN I need to report on how many specimens have been imaged

THEN I need to search collection images by the 'date taken'

AND the result needs to be clearly presented and this 'date taken' data needs to be clearly accessed.

Please rename the "Date Original Version Created" field to "Date Taken" this naming will be consistent with what is viewed in the image file details and more clearly communicate the data contained in the field.

I will be consulting the collection managers as to their requirements on viewing this date and the other dates currently displayed for uploaded objects.

**Comment by cgendreau** [comment]
But Date Taken is quite image centric. The object-store can store objects including documents. Date Taken of a document can be confusing. The Term in the standard is DigitizationDate: https://tdwg.github.io/ac/termlist/#ac_digitizationDate .

**Comment by jmacklin** [comment]
Agreed, this makes more sense as a generic solution. I think digitization date is very clear as a label. For searching, we can use digitization date and file type(s) filters to fulfill Heather's use case above.

**Comment by heathercole** [comment]
Not clear to me which type of other file-types would be relevant to record "digitization" date. While there are documents included, for many (most) the date they were "scanned" does not seem relevant for management purposes. I already have this on my list of topics to review with the collection managers.

**Comment by cgendreau** [comment]
sounds, movies, scan of a drawing

**Comment by jmacklin** [comment]
These are good examples. Agreed that management likely will not need a report for some other file types but the key point here is that we are looking for a generic solution. This is why this term was created in Audubon Core.

**Comment by heathercole** [comment]
it will be worth paying extra attention to what date information is used for 'digitization date' for different files types, scans that create 'images' I think still generate a 'date taken', but .mp4s do not, they do have a 'media created' date field, but haven't investigated that much, though it does seem more informative than the 'date created' and 'date modified' info which relate to the particular copy of the file (vs. subject/object itself)

**Comment by cgendreau** [comment]
Just to clarify, 'date created' and 'date modified' are administrative metadata about the record, not the file.

Date created is when the record was created in the object-store and date modified represents when its data was last updated. It is not linked with the file but the record representing the file.

**Comment by heathercole** [comment]
ach, so many dates, in the new year I will review with the CMs to clarify requirements and terminology so that the fields and titles are as self-explanatory and clear as possible.

**Comment by rintoult** [comment]
I feel like using the phrase "Date Taken" only applies to photos, so would be strange wording for other objects that will get uploaded. as I think you mention above. I don't know where other types of files like PDFs save that kind of metadata - especially when they are getting moved around.

**Comment by heathercole** [comment]
i am reviewing this field in terms of 'date taken' vs. 'digitized date', and consulting the collection manager's about their requirements related to this. will provide feedback asap
